### DISCUSSION TITLE
create-next-app fails with error Command failed with exit code 1.

### DISCUSSION THREAD

**Discussion by pkail** [vercel/next.js#12332] [Help]
I typed create-next-app into the terminal.  When it had downloaded everything, I typed yarn dev as described here: https://nextjs.org/docs/getting-started
I also tried the 'manual install' several times.  I deleted the entire directory in between each attempt.

## To Reproduce
Follow the instructions here: https://nextjs.org/docs/getting-started

## Expected behavior
I expected a working app. 

A working app.  A build that worked.  A project that compiled correctly.
## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of Next.js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]
Linux Mint 19.2 64-bit.     
next: "9.3.6",
node 13.2.0
## Additional context
It's got something to do with not being able to find autoprefixer.  

>>Couldn't someone have tested it before putting it online?  I have lost all faith in next.js.

Error: Cannot find module 'autoprefixer'
Require stack:
- /home/p/Data/WS/postcss.config.js
- /home/p/Data/WS/tlc/node_modules/next/dist/lib/find-config.js
- /home/p/Data/WS/tlc/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js
- /home/p/Data/WS/tlc/node_modules/next/dist/build/webpack/config/blocks/css/index.js
- /home/p/Data/WS/tlc/node_modules/next/dist/build/webpack/config/index.js
- /home/p/Data/WS/tlc/node_modules/next/dist/build/webpack-config.js
- /home/p/Data/WS/tlc/node_modules/next/dist/server/hot-reloader.js
- /home/p/Data/WS/tlc/node_modules/next/dist/server/next-dev-server.js
- /home/p/Data/WS/tlc/node_modules/next/dist/server/next.js
- /home/p/Data/WS/tlc/node_modules/next/dist/server/lib/start-server.js
- /home/p/Data/WS/tlc/node_modules/next/dist/cli/next-dev.js
- /home/p/Data/WS/tlc/node_modules/next/dist/bin/next
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:954:17)
    at Function.Module._load (internal/modules/cjs/loader.js:847:27)
    at Module.require (internal/modules/cjs/loader.js:1016:19)
    at require (internal/modules/cjs/helpers.js:69:18)
    at Object.<anonymous> (/home/p/Data/WS/postcss.config.js:2:13)
    at Module._compile (internal/modules/cjs/loader.js:1121:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1160:10)
    at Module.load (internal/modules/cjs/loader.js:976:32)
    at Function.Module._load (internal/modules/cjs/loader.js:884:14)
    at Module.require (internal/modules/cjs/loader.js:1016:19)
    at require (internal/modules/cjs/helpers.js:69:18)
    at findConfig (/home/p/Data/WS/tlc/node_modules/next/dist/lib/find-config.js:7:181)
    at async getPostCssPlugins (/home/p/Data/WS/tlc/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js:4:141)
    at async __overrideCssConfiguration (/home/p/Data/WS/tlc/node_modules/next/dist/build/webpack/config/blocks/css/overrideCssConfiguration.js:1:272)
    at async getBaseWebpackConfig (/home/p/Data/WS/tlc/node_modules/next/dist/build/webpack-config.js:118:120)
    at async Promise.all (index 0) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/p/Data/WS/postcss.config.js',
    '/home/p/Data/WS/tlc/node_modules/next/dist/lib/find-config.js',
    '/home/p/Data/WS/tlc/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js',
    '/home/p/Data/WS/tlc/node_modules/next/dist/build/webpack/config/blocks/css/index.js',
    '/home/p/Data/WS/tlc/node_modules/next/dist/build/webpack/config/index.js',
    '/home/p/Data/WS/tlc/node_modules/next/dist/build/webpack-config.js',
    '/home/p/Data/WS/tlc/node_modules/next/dist/server/hot-reloader.js',
    '/home/p/Data/WS/tlc/node_modules/next/dist/server/next-dev-server.js',
    '/home/p/Data/WS/tlc/node_modules/next/dist/server/next.js',
    '/home/p/Data/WS/tlc/node_modules/next/dist/server/lib/start-server.js',
    '/home/p/Data/WS/tlc/node_modules/next/dist/cli/next-dev.js',
    '/home/p/Data/WS/tlc/node_modules/next/dist/bin/next'
  ]
}
error Command failed with exit code 1.

**Comment by pkail** [comment]
There are errors going back to 2018 with autoprefixer.  It doesn't work properly in any of the tutorial videos on YouTube.  I have tried looking at various solutions on SE, but everything is for much older versions, and doesn't seem to work even for them.

**Comment by timneutkens** [comment]
Looks like you have a global postcss.config.js:

```
/home/p/Data/WS/postcss.config.js
```

Which imports a module that you don't have installed. Probably you didn't intent to have a postcss configuration file in your home directory so I'd recommend deleting the file.

The getting started guide works fine and we have extensive integration tests to ensure Next.js doesn't break between versions.

**Comment by pkail** [comment]
It does work without the config file.  But Next put it there, not me.

**Reply by timneutkens** [reply]
A postcss.config.js file is not created by Next.js, it's only read for configuration.

**Comment by pkail** [comment]
That's a mystery.

**Comment by pkail** [comment]
Anyway, thanks for your help.
